docs/index.rst

```diff
@@ -0,0 +1,21 @@
+.. Python Zeppelin documentation master file, created by
+   sphinx-quickstart on Mon May 22 13:57:54 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Python Zeppelin's documentation!
+===========================================
+Python Zeppelin is a utility that converts native Zeppelin notebooks into Markdown format.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
```

Comment:
please add `zeppelin` to this toctree otherwise it won't be reachable from the homepage.